### PR TITLE
Add `csp-headers` command

### LINKED ISSUE #11: Add command to output headers
It would be great if we could add a `csp-headers` command to `ember` to make copying the CSP-headers ridiculously easy. So I could just do:

```
ember csp-headers --environment production
```

And get the right headers for my production environment.

### PR DESCRIPTION
**Pull request by avdv**
Fixes #11.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -187,3 +187,52 @@ module.exports = {
...
+        ],
+
+        run: function(options) {
+          //this.ui.writeLine(JSON.stringify(options));
+          var path = require('path');
+          var fs = require('fs');
+          var merge = require('merge');
+          var env = options.environment;
+
+          // FIXME: how do I read the app config?
```
Comment (rwjblue) [inline index.js]: `this.project.config(options.environment)`

File: index.js [hunk 2]
```diff
@@ -187,3 +187,52 @@ module.exports = {
...
+          var configPath = path.join(this.project.root, 'config/environment.js');
+
+          // FIXME: place the default config somewhere else so we can read it in the command?
```
Comment (rwjblue) [inline index.js]: Ya, that seems good

**Comment by rwjblue** [comment]
Overall this looks good. I generally would prefer to move the commands to `lib/commands.js` and have the `includedCommands` hook look something like this:

```js
includedCommands: function() {
  return require('./commands');
}
```

Then have a `lib/commands.js` with the command added here.

**Comment by avdv** [comment]
Thanks! I have implemented your changes, looks good already.

The only thing is, that the output is intermingled with some warnings and the "there's a new ember-cli version" message:

```
(node:31478) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
(node:31478) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
WARNING: WARNING: Node v6.5.0 has currently not been tested against Ember CLI and may result in unexpected behaviour.
version: 2.4.1

A new version of ember-cli is available (2.8.0).
default-src 'none'; script-src 'self'; font-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; media-src 'self'; report-uri /csp-report;
```
Is there a way to reliably disable any output except for the generated headers?

Otherwise I probably introduce an option to write into a given file...

**Comment by rwjblue** [comment]
I would add some preamble prose a few new lines and the emit the actual headers (that makes it nice for human use), then start testing with the `--silent` flag and make sure it only emits the headers (which would make it nice for usage in shell pipelines).

**Comment by avdv** [comment]
Alright, using `node ~/apps/node/bin/ember csp-headers --silent --environment production` produces the desired output.

**Comment by avdv** [comment]
ping. Any objections? WDYT?

**Comment by sandstrom** [comment]
@avdv Looks good to me! Nice work! I'll leave it to @rwjblue for a final decision though.

**Comment by rwjblue** [comment]
I'll try to review again this week (sorry for delay)

**Comment by avdv** [comment]
polite ping -- could you have a look again, thanks!

**Comment by avdv** [comment]
@rwjblue /ping

(note that I have switched to an exponential backoff ping algorithm to avoid flooding your mailbox :wink:)

**Comment by rwjblue** [comment]
Sorry about the super long delay here!

**Pull request merged**